### DISCUSSION TITLE
Talent Builder - Filters don't affect replay list

### DISCUSSION THREAD

**Discussion by AlternatiRP** [Heroes-Profile/heroesprofile#915] [Ideas]
**Describe the bug**
Filters like Player/Hero/Role rank or Hero Level dont work correctly. Like if u put Master player/hero/role u dont get a player with such player/hero/role mmr.

**To Reproduce**
1. Go to [Global Talent Builder](https://www.heroesprofile.com/Global/Talents/Builder/Alarak?timeframe_type=minor&timeframe=2.55.4.91769,2.55.4.91418,2.55.4.91368,2.55.3.91093,2.55.3.91081,2.55.3.91020,2.55.3.90670,2.55.3.89754&game_type=sl&region=NA,EU&game_map=Battlefield%20of%20Eternity&league_tier=Master&hero_league_tier=Master&role_league_tier=Master&mirror=0)
2. Select these talents [T1021100,Alarak] and click on **Scroll To Builds**
3. Scroll down to **Builds**
4. Click the [first game](https://www.heroesprofile.com/Match/Single/52322236)

**UnExpected behavior**
I get the player on Alarak who has lower mmr than was filtered

**Screenshots**

![image](https://github.com/Heroes-Profile/heroesprofile/assets/107478063/c4a2778a-031a-4cfe-85e6-b44685aeac48)
![image](https://github.com/Heroes-Profile/heroesprofile/assets/107478063/f450642d-17ff-404f-86df-cbc800f44237)

**Desktop (please complete the following information):**
 - OS: Win
 - Browser: Chrome
 - Version: 107.0.5304.107

**Comment by Zemill** [comment]
We have a notification above that section

*Possible Replays do not take into account Hero Level, Player Rank, Hero Rank, Role Rank, or Mirror Match Filter options


I am going to put this as an enhancement

**Comment by AlternatiRP** [comment]
Thx. How long will it take btw?

**Comment by Zemill** [comment]
> Thx. How long will it take btw?

I do not know honestly, while it seems simple enough, there is some difficulty with the representation of ranks and how they are stored relative to the global cache data.

Getting it for everything other than ranks might be easy, and I might be able to get a not perfect version working for ranks.  It will  have to be after this next release I am working on now

**Comment by AlternatiRP** [comment]
thx

**Comment by Zemill** [comment]
This will be added back in when https://github.com/Heroes-Profile/heroesprofile/pull/1071 is merged
